sherpa/astro/ui/serialize.py

```diff
@@ -654,7 +654,7 @@ def tostatement(key, val):
         # is probably what we want but is a change, so leave
         # for now.
         # return 'set_iter_method_opt("{}", {})'.format(key, val)
```

Comment:
This comment can go away. No need to keep commented return statements around.

Reply:
I've actually made a small change to the existing code - use of a f-string - and cleaned up the comment line (to point out a difference to a different `tostatement` routine in this module). There should be no significant functional change.